RestApi/Sepes.RestApi/Model/PodInput.cs

```diff
@@ -15,19 +16,9 @@ public class PodInput
 
         public Pod ToPod()
         {
-            var podIncoming = new List<Rule>();
-            var podOutgoing = new List<Rule>();
-            var podUsers = new List<User>();
-            
-            foreach (RuleInput rule in incoming) {
-                podIncoming.Add(rule.ToRule());
-            }
-            foreach (RuleInput rule in outgoing) {
-                podOutgoing.Add(rule.ToRule());
-            }
-            foreach (string user in users) {
-                podUsers.Add(new User("", user, ""));
-            }
+            var podIncoming = incoming != null ? incoming.Select(r => r.ToRule()) : new List<Rule>();
+            var podOutgoing = outgoing != null ? outgoing.Select(r => r.ToRule()) : new List<Rule>();
```

Comment:
Not a big fan of rewriting stuff to reduce line count at cost of readability

Reply:
I personally find if statement to be a lot more readable than ?: Like the .Select
Also we have ?. and ?? in c# that handle values that may be null.
So could be written as: `var podIncoming = incoming?.Select(r => r.ToRule()) ?? new List<Rule>();` or even `var podIncoming = incoming?.Select(r => r.ToRule()) ?? default;` if the compiler can figure out the type on its own.

Another way is to add `public RuleInput[] incoming { get; set; } = new RuleInput[];` At line 12. As that will sett the default value to and empty array and not null. And then you don't need the null check in to pod. Then you only need the `var podIncoming = incoming.Select(r => r.ToRule());`